### ISSUE TITLE
Got error when installing the GiottoClass on Rocky Linux

### ISSUE THREAD

**Issue by shaojunyu** [drieslab/GiottoClass#214]
Hi,

I encountered an error while installing the GiottoClass on Rocky Linux, but I was able to successfully install it on my Mac. Here are the details of the error:

- install command
```bash
library(devtools)
install_github('drieslab/GiottoClass')
```

Error:
> Error: extract-methods.Rd:210: no comma in argument list following \S4method

Error log:

> Downloading GitHub repo drieslab/GiottoClass@HEAD
> ── R CMD build ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
> ✔  checking for file ‘/tmp/RtmpC8LrFR/remotes1e7e91ec18380/drieslab-GiottoClass-848d41d/DESCRIPTION’ ...
> ─  preparing ‘GiottoClass’:
> ✔  checking DESCRIPTION meta-information ...
> ─  checking for LF line-endings in source and make files and shell scripts (563ms)
> ─  checking for empty or unneeded directories
> ─  building ‘GiottoClass_0.3.2.tar.gz’
> 
> Installing package into ‘/home/user/R/x86_64-pc-linux-gnu-library/4.4’
> (as ‘lib’ is unspecified)
> * installing *source* package ‘GiottoClass’ ...
> ** using staged installation
> ** R
> ** inst
> ** byte-compile and prepare package for lazy loading
> in method for ‘as.sp’ with signature ‘"sf"’: no definition for class “sf”

> ....

> Error: extract-methods.Rd:210: no comma in argument list following \S4method
> * removing ‘/home/user/R/x86_64-pc-linux-gnu-library/4.4/GiottoClass’
>Warning message:
> In i.p(...) :
>  installation of package ‘/tmp/RtmpC8LrFR/file1e7e9586b98ff/GiottoClass_0.3.2.tar.gz’ had non-zero exit status

- sessionInfo
```r
> sessionInfo()
R version 4.4.1 (2024-06-14)
Platform: x86_64-pc-linux-gnu
Running under: Rocky Linux 8.10 (Green Obsidian)

Matrix products: default
BLAS/LAPACK: /usr/lib64/libopenblasp-r0.3.15.so;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

time zone: UTC
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] devtools_2.4.5 usethis_3.0.0

loaded via a namespace (and not attached):
 [1] miniUI_0.1.1.1    compiler_4.4.1    promises_1.3.0    Rcpp_1.0.13
 [5] stringr_1.5.1     callr_3.7.6       later_1.3.2       fastmap_1.2.0
 [9] mime_0.12         R6_2.5.1          pak_0.7.2         curl_5.2.1
[13] htmlwidgets_1.6.4 desc_1.4.3        profvis_0.3.8     shiny_1.9.0
[17] rlang_1.1.4       cachem_1.1.0      stringi_1.8.4     httpuv_1.6.15
[21] fs_1.6.4          pkgload_1.4.0     memoise_2.0.1     cli_3.6.3
[25] magrittr_2.0.3    ps_1.7.7          processx_3.8.4    digest_0.6.36
[29] xtable_1.8-4      remotes_2.5.0     lifecycle_1.0.4   vctrs_0.6.5
[33] glue_1.7.0        urlchecker_1.0.1  sessioninfo_1.2.2 pkgbuild_1.4.4
[37] purrr_1.0.2       tools_4.4.1       ellipsis_0.3.2    htmltools_0.5.8.1
```

Thanks for your help!

**Comment by jiajic** [comment]
Hi, thanks for reporting this!
I'm not too sure what is going wrong here. It appears that somehow building the documentation is breaking the install.
Could you try reinstalling _roxygen2_ on Linux and seeing if that fixes it? I am on _roxygen2 7.3.2_

Thanks
George

**Comment by shaojunyu** [comment]
I have tried reinstalling roxygen2 (v 7.3.2), but the issue persists. I also attempted to install the [latest commit](https://github.com/drieslab/GiottoClass/commit/e622a46c52aa9d720aaaf3ccc1912658a6aaa31d) of this tool, but encountered the same error. It is puzzling because I tested the installation with R 4.4.1 on three different operating systems: Mac OS Sonoma, Debian 12.6, and Rocky 8.10. This issue only occurs for R 4.4.1 on Rocky Linux; it works fine on the other two platforms.
